game-core/src/main/java/org/triplea/game/server/HeadlessGameServer.java

```diff
@@ -580,19 +567,11 @@ public Chat getChat() {
    * </p>
    */
   public static void start(final String[] args) {
-    final ArgValidationResult validation = HeadlessGameServerCliParam.validateArgs(args);
-    if (!validation.isValid()) {
-      log.severe(String.format("Failed to start, improper args: %s\n"
-          + "Errors:\n- %s\n"
-          + "Example usage: %s",
-          Arrays.toString(args),
-          String.join("\n- ", validation.getErrorMessages()),
-          HeadlessGameServerCliParam.exampleUsage()));
-      return;
-    }
-
     ClientSetting.initialize();
+    System.setProperty(LOBBY_GAME_COMMENTS, "automated_host");
```

Comment:
Is the intention here to lock down these properties or just to provide defaults?  If the former, these calls probably need to be after `ArgParser.handleCommandLineArgs(args)` to prevent a user from specifying different values on the command line.

Reply:
Motivation is:
1. Simplify/reduce, best thing I can think to do before seriously looking at environment variables for secrets or other configuration
2. Make it easier to run bots

Locking down is only going to be so successful given the hosting user of a bot has access to the source code. Fail-early still applies though for bad/missing parameters of course, that probably should be the goal of the validation done here.